worker-workflow/src/main/java/com/github/cafdataprocessing/WorkflowProcessingScripts.java

```diff
@@ -42,9 +42,9 @@ public static void setScripts(final Document document, final String workflowAsJa
         throws ScriptException
     {
         // Add temporary script to the task using the setScriptInline setter.
-        setWorkflowProcessingScripts(document, workflowAsJavaScript, ScriptType.InlineScript, "temp-workflow.js", false);
+        setWorkflowProcessingScripts(document, workflowAsJavaScript, ScriptType.InlineScript, "temp-workflow.js");
         // Add persistant script to the task using the setScriptByReference setter.
-        setWorkflowProcessingScripts(document, workflowAsJavaScript, ScriptType.StorageReference, "workflow.js", true);
+        setWorkflowProcessingScripts(document, workflowAsJavaScript, ScriptType.StorageReference, "workflow.js");
```

Comment:
Shouldn't the parameter here be `workflowStorageRef`?

Reply:
Yes, updated now.